tests/ecosystem/upgrade/test_upgrade_ocp.py

```diff
@@ -159,6 +161,14 @@ def test_upgrade_ocp(self, reduce_and_resume_cluster_load):
         cluster_ver = ocp.run_cmd("oc get clusterversions/version -o yaml")
         logger.debug(f"Cluster versions post upgrade:\n{cluster_ver}")
 
+        # Login to OCP console and run ODF dashboard validation check
+        version_post_upgrade = parse_version(ocp_upgrade_version)
+        ocs_version = version.get_semantic_version(version="4.9", only_major_minor=True)
+        if version_post_upgrade >= ocs_version:
```

Comment:
```suggestion
       
        if version_post_upgrade >= version.VERSION_4_9:
```

Reply:
> 

I am getting this error again:

2022-11-25 16:50:10  >       if version_post_upgrade >= version.VERSION_4_9:
2022-11-25 16:50:10  E       TypeError: '>=' not supported between instances of 'LegacyVersion' and 'Version'
2022-11-25 16:50:10  
2022-11-25 16:50:10  tests/ecosystem/upgrade/test_upgrade_ocp.py:166: TypeError